src/app/config.service.server.ts

```diff
@@ -1,7 +1,7 @@
 import {Inject, Injectable, PLATFORM_ID} from '@angular/core';
 import {makeStateKey, TransferState} from '@angular/platform-browser';
 declare function require(name: string);
-const config = require('../../config');
+const configFile = require('../../config');
```

Comment:
not sure how i feel about the name change.

Reply:
I was not happy about it also but the file had shadow names and for it was best idea... have a look at the file... it a puzzle :D